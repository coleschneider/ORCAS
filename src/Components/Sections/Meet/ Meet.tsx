import * as React from 'react';
import Carousel from 'Components/Carousel/Carousel';
import * as PlaceholderAvatar from 'Images/avatar.png';
import { createSection } from 'utils/sectionUtils';
const MeetWrapper = createSection({
  className: 'about-how',
  displayTitle: 'Meet the Seniors!',
  sectionName: 'seniors',
});
export const Meet = () => (
  <MeetWrapper>
    <Carousel>
      <Carousel.Slide author={'auser1'} avatar={PlaceholderAvatar}>
        Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi
        eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      </Carousel.Slide>
      <Carousel.Slide author={'auser2'} avatar={PlaceholderAvatar}>
        Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi
        eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      </Carousel.Slide>
      <Carousel.Slide author={'auser3'} avatar={PlaceholderAvatar}>
        Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi
        eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      </Carousel.Slide>
    </Carousel>
  </MeetWrapper>
);

export default Meet;
