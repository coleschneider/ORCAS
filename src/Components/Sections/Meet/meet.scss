@import '.../../../../../stylesheets/layout/colors.scss';

.testimonials {
  display: flex;
   padding: 16px;

  @media only screen and (max-width:800px) {
    flex-direction: column;
  }
}

.testimonial-container {
  padding: 16px;
  margin-left: 15px;
  margin-right: 15px;
  border-radius: 5px;
  background-color: $bg-grey;
  @media only screen and (max-width:800px) {
    margin-bottom: 15px;
  }
}

.s-senior-testimonials {
  padding-bottom: 16rem;
  padding-top: 5rem;
  background: $blue-primary
  
}

.s-senior-testimonials .display-1,
.s-senior-testimonials h3 {
  color: black;
}
.s-senior-testimonials p.lead
{
  color: #ffffff;
}