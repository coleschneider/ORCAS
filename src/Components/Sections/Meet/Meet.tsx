import * as React from 'react';
import Carousel from 'Components/Carousel/Carousel';
import * as PlaceholderAvatar from 'Images/avatar.png';
import './meet.scss';
import { createSection } from 'utils/sectionUtils';
const TeamWrapper = createSection({
  className: 'about-how',
  displayTitle: 'Meet the Team!',
  sectionName: 'team',
});
const TestimonialWrapper = createSection({
  className: 'senior-testimonials',
  description: 'Hear the stories...',
  displayTitle: 'Meet the Seniors!',
  sectionName: 'seniors',
});

export const MeetTheSeniors = () => (
  <TestimonialWrapper>
    <div className="testimonials">
      <div className="testimonial-container">
        <span className="quote">❝</span>
        Excepturi nam cupiditate culpa doloremque deleniti repellat.
        <span className="quote-2">❞</span>
        <br />
        -A User
      </div>
      <div className="testimonial-container">
        <span className="quote">❝</span>
        Excepturi nam cupiditate culpa doloremque deleniti repellat.
        <span className="quote-2">❞</span>
        <br />
        -A User
      </div>
      <div className="testimonial-container">
        <span className="quote">❝</span>
        Excepturi nam cupiditate culpa doloremque deleniti repellat.
        <span className="quote-2">❞</span>
        <br />
        -A User
      </div>
    </div>
  </TestimonialWrapper>
);

const UserSlide = ({ avatar, role, children, author }) => (
  <div className="meet__slide">
    <img alt="Author image" src={avatar} className="meet__avatar" />
    <p>{children}</p>
    <div className="meet__author">
      <span className="meet__name">{author}</span>
      <a href="#0" className="meet__link">
        - {role}
      </a>
    </div>
  </div>
);
export const MeetTheTeam = () => (
  <TeamWrapper>
    <Carousel>
      <UserSlide author="Kay Brainard" role="President" avatar={PlaceholderAvatar}>
        Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi
        eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      </UserSlide>
      <UserSlide author="Alison O’Neill" role="Vice-President" avatar={PlaceholderAvatar}>
        Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi
        eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      </UserSlide>
      <UserSlide author="Lori Nasatir" role="Second-In-Command" avatar={PlaceholderAvatar}>
        Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi
        eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      </UserSlide>
      <UserSlide author="Barbara Page" role="Publicity Contact" avatar={PlaceholderAvatar}>
        Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi
        eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      </UserSlide>
      <UserSlide author="Kathie Lanker" role="Member" avatar={PlaceholderAvatar}>
        Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi
        eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      </UserSlide>
      <UserSlide author="Michael Williams" role="Member" avatar={PlaceholderAvatar}>
        Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi
        eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      </UserSlide>

      <UserSlide author="Clark Owens" role="Member" avatar={PlaceholderAvatar}>
        Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi
        eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      </UserSlide>
    </Carousel>
  </TeamWrapper>
);
