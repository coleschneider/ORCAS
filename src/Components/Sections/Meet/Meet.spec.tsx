import * as React from 'react';
import Meet from 'Sections/Meet/ Meet';

describe('Meet', () => {
  it('should match the screenshot', () => {
    const wrapper = shallow(<Meet />);
    expect(wrapper).toMatchSnapshot();
  });
});
