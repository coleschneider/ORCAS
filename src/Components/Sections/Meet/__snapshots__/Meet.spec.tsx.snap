// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meet MeetTheSeniors should match the screenshot 1`] = `
<Component>
  <div
    className="testimonials"
  >
    <div
      className="testimonial-container"
    >
      <span
        className="quote"
      >
        ❝
      </span>
      Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      <span
        className="quote-2"
      >
        ❞
      </span>
    </div>
    <div
      className="testimonial-container"
    >
      <span
        className="quote"
      >
        ❝
      </span>
      Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      <span
        className="quote-2"
      >
        ❞
      </span>
    </div>
    <div
      className="testimonial-container"
    >
      <span
        className="quote"
      >
        ❝
      </span>
      Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      <span
        className="quote-2"
      >
        ❞
      </span>
    </div>
    <div
      className="testimonial-container"
    >
      <span
        className="quote"
      >
        ❝
      </span>
      Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
      <span
        className="quote-2"
      >
        ❞
      </span>
    </div>
  </div>
</Component>
`;

exports[`Meet MeetTheTeam should match the screenshot 1`] = `
<Component>
  <Carousel>
    <Component
      author="Kay Brainard"
      avatar={
        Object {
          "0": "a",
          "1": "v",
          "2": "a",
          "3": "t",
          "4": "a",
          "5": "r",
          "6": ".",
          "7": "p",
          "8": "n",
          "9": "g",
          "default": "avatar.png",
        }
      }
      role="President"
    >
      Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
    </Component>
    <Component
      author="Alison O’Neill"
      avatar={
        Object {
          "0": "a",
          "1": "v",
          "2": "a",
          "3": "t",
          "4": "a",
          "5": "r",
          "6": ".",
          "7": "p",
          "8": "n",
          "9": "g",
          "default": "avatar.png",
        }
      }
      role="Vice-President"
    >
      Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
    </Component>
    <Component
      author="Lori Nasatir"
      avatar={
        Object {
          "0": "a",
          "1": "v",
          "2": "a",
          "3": "t",
          "4": "a",
          "5": "r",
          "6": ".",
          "7": "p",
          "8": "n",
          "9": "g",
          "default": "avatar.png",
        }
      }
      role="Second-In-Command"
    >
      Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
    </Component>
    <Component
      author="Barbara Page"
      avatar={
        Object {
          "0": "a",
          "1": "v",
          "2": "a",
          "3": "t",
          "4": "a",
          "5": "r",
          "6": ".",
          "7": "p",
          "8": "n",
          "9": "g",
          "default": "avatar.png",
        }
      }
      role="Publicity Contact"
    >
      Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
    </Component>
    <Component
      author="Kathie Lanker"
      avatar={
        Object {
          "0": "a",
          "1": "v",
          "2": "a",
          "3": "t",
          "4": "a",
          "5": "r",
          "6": ".",
          "7": "p",
          "8": "n",
          "9": "g",
          "default": "avatar.png",
        }
      }
      role="Member"
    >
      Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
    </Component>
    <Component
      author="Michael Williams"
      avatar={
        Object {
          "0": "a",
          "1": "v",
          "2": "a",
          "3": "t",
          "4": "a",
          "5": "r",
          "6": ".",
          "7": "p",
          "8": "n",
          "9": "g",
          "default": "avatar.png",
        }
      }
      role="Member"
    >
      Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
    </Component>
    <Component
      author="Clark Owens"
      avatar={
        Object {
          "0": "a",
          "1": "v",
          "2": "a",
          "3": "t",
          "4": "a",
          "5": "r",
          "6": ".",
          "7": "p",
          "8": "n",
          "9": "g",
          "default": "avatar.png",
        }
      }
      role="Member"
    >
      Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci. Nisi eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
    </Component>
  </Carousel>
</Component>
`;
